### ISSUE TITLE
Flagging Active and Inactive Mailing Lists

### ISSUE THREAD

**Issue by sharanf** [apache/incubator-ponymail#471]
Some of the older ASF mailing lists are no longer in use and are only there for display purposes. In theory would we not want to allow users to start any new threads or have users request subscriptions to them.  

This enhancement is to look at adding a  flag to a mailing to indicate if the list is active (e.g. allowing new posts and subscription requests) or inactive (not allowing new posts or subscription requests).

- If the list is active then the standard buttons to start a new thread, subscribe or unsubscribe to the list are displayed.
- If the list is inactive instead of displaying the standard buttons for starting a new thread or subscribing, a message could be displayed saying that posts and subscriptions to the list are closed for the list.

**Comment by sebbASF** [comment]
The ES database currently contains a mailinglists mapping which could probably be extended to include extra details such as the date of closure.
At present it only seems to be used to hold MailMan list info (if present)

**Comment by Humbedooh** [comment]
There isn't really a way for PM to know if a list is active or inactive other than whether it is getting emails. We *could* have some sort of tests for activity and suggest that a list *may* be inactive if below a certain threshold, but doing more than that would require either manual intervention (perhaps a CLI script to mark lists?) or tying it into the mailing list service itself.

**Comment by sebbASF** [comment]
I agree that list activity cannot be used to determine whether a list is still active or not.
Some lists have very little activity, and a list with recent activity may have been closed down.

There may be other reasons for disallowing posting, e.g. for list archives that are aliases.

**Comment by sebbASF** [comment]
Regardless of how the list state is determined, there should be a way to record it, and for PM to take note of it in displays etc.
